iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/infospace/ImportInformationSpaceJob.java

```diff
@@ -416,6 +457,9 @@ private static String outputPathFor(String path, String subPath) {
         
         @Parameter(names = "-outputNameProjectOrganization", required = true)
         private String outputNameProjectOrganization;
+
+        @DynamicParameter(names = "-D", description = "dynamic parameters related to document metadata truncation")
+        private Map<String, String> documentMetadataTruncationParams = new HashMap<>();
```

Comment:
The parameters are definitely not dynamic, it would be clearer to define them as normal parameters. I thought that the reason to use dynamic parameters was to keep `ImportInformationSpaceJobParameters` small and clean, but if we can accept a few more fields in this class then I'd go with adding explicit parameters. 

Looking a bit in the future if we find that we need to add even more truncation parameters I'd introduce a dedicated workflow step that would be responsible for truncation. This step would take all the needed parameters as explicit parameters to the job.

Reply:
Or maybe you think we should add a dedicated workflow step now ?

Reply:
I think we don't need to go that far, I mean I don't expect much more truncation-related parameters to pop-up.